### PR TITLE
Feat112

### PR DESCRIPTION
**Pull request by davidlewallen**

### CODE REVIEW

**Comment by ktorng** [comment]
so broke

File: client/actions/actions.js [hunk 1]
```diff
@@ -161,6 +163,34 @@ export function createEvent(formData, currentUser) {
...
+export function editEvent(currentEvent) {
+  return {
+    type: EDIT_EVENT,
+    payload: currentEvent
+  }
+}
+
+export function updateEvent(updatedEvent, currentUser, eventId) {
+  const url = `/api/events/${eventId}`
+  const body = {
```
Comment (ktorng) [inline client/actions/actions.js]: const body = updatedEvent;

File: client/reducers/reducer_currentEvent.js [hunk 2]
```diff
@@ -3,7 +3,9 @@ import {
   JOIN_EVENT, 
   HIDE_EVENT, 
   SET_CURRENT_EVENT, 
-  LEAVE_EVENT
+  LEAVE_EVENT,
+  EDIT_EVENT,
```
Comment (ktorng) [inline client/reducers/reducer_currentEvent.js]: delete

File: client/actions/actions.js (hunk 1 above)
Comment (davidlewallen) [inline client/actions/actions.js]: Fixed

File: client/reducers/reducer_currentEvent.js (hunk 2 above)
Comment (davidlewallen) [inline client/reducers/reducer_currentEvent.js]: Fixed

**Pull request merged**